File: scala/001/Main.scala
/*
 * ProjectEuler 001
 * Scala / Main.scala
 * copyright (c) 2015 Susisu
 *
 * Scala 2.11.4
 * `scalac Main.scala -d bin`
 */

object Main {
    def main(args: Array[String]) {
        var sum = 0;
        for (n <- 1 until 1000) {
            if (n % 3 == 0 || n % 5 == 0) {
                sum += n;
            }
        }
        println(sum);
    }
}

File: python/001/main.py
# ProjectEuler 000
# Python / main.py
# copyright (c) 2015 Susisu
#
# Python 3.4.2
# `python3 main.py`


def main():
    sum = 0
    for n in range(1, 1000):
        if n % 3 == 0 or n % 5 == 0:
            sum += n
    print(sum)

main()
